sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -191,6 +197,12 @@ static rom_error_t mask_rom_verify(const manifest_t *manifest,
   // Verify signature
   hmac_digest_t act_digest;
   hmac_sha256_final(&act_digest);
+
+  static_assert(sizeof(boot_measurements.rom_ext.data) == sizeof(act_digest.digest), "Unexpected ROM_EXT digest size.");
+  for (size_t i = 0; i < ARRAYSIZE(act_digest.digest); ++i) {
+    boot_measurements.rom_ext.data[i] = act_digest.digest[i];
+  }
```

Comment:
nit: Not sure if this matters but we could probably write directly to the field in the static critical section instead of copying from `act_digest`. The type of the field cannot be keymgr_binding_value though.